openmdao/visualization/n2_viewer/n2_viewer.py

```diff
@@ -32,6 +33,7 @@
 from openmdao import __version__ as openmdao_version
 
 _MAX_ARRAY_SIZE_FOR_REPR_VAL = 1000  # If var has more elements than this do not pass to N2
+_MAX_OPTION_SIZE = int(1e7)          # If option value is bigger than this do not pass to N2
```

Comment:
I wonder if it should be even smaller, since our max array size is only 1000 elements.  I guess this is bytes though; I am not sure how many bytes are in a 1000 element array in numpy.

Reply:
yeah.. I set it to the absolute max order of magnitude before it would cause an encoding error...  we could definitely set it to a more practical value, since large values are pretty useless visually in the n2 anyway...

```
>>> k.shape
(1000,)
>>> sys.getsizeof(k)
8112
>>> j.shape
(1000, 1000)
>>> sys.getsizeof(j)
8000128
```

Reply:
I changed it to `1e4`..  because larger values in the N2 don't really make sense, per above, and because a user may inadvertently and unnecessarily incur the performance overhead of encoding the larger value for no good reason.